src/Plugin/Double.php

```diff
@@ -101,7 +101,13 @@ public static function classname($options = [])
             $code = static::generate($options);
             $nodes = $parser::parse($code);
             $code = $parser::unparse(static::$_pointcut->process($nodes));
+
+            $iniGet = ini_get("error_reporting");
+            ini_set("error_reporting", E_ALL & ~E_DEPRECATED);
+
             eval('?>' . $code);
+
+            ini_set("error_reporting", $iniGet);
```

Comment:
@jails this disable `E_DEPRECATED` temporary for eval due generation of code that produce requirement of `#[\ReturnTypeWillChange]`, do you have better solution?

Reply:
@jails I updated with `preg_replace` so it replace on `(public( static)? function)` generation https://github.com/kahlan/kahlan/pull/392/commits/4fc80b43b516824d8ce2f82191d86cf93e238ab8